src/main/java/com/example/demo/controllers/LocationController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo.controllers;
+
+import com.example.demo.entities.LocationEntity;
+import com.example.demo.model.dto.LocationCreationDTO;
+import com.example.demo.services.LocationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/location")
+public class LocationController {
+
+    @Autowired
+    LocationService locationService;
+
+    @PostMapping(path = "/createLocation")
+    public LocationEntity createLocation(@RequestBody LocationCreationDTO locationDTO)
+    {
+        //include validation possibly.
+        LocationEntity location = locationDTO.convertDtoToEntity();
+        return locationService.createLocation(location);
+    }
+
+    @ResponseBody
+    @GetMapping(path = "/findLocation")
```

Comment:
Because you're using HTTP GET, that implies 'find', 'fetch' type verbs. So you'd also just use /location here